test/parser/variables.spec.js

```diff
@@ -14,10 +14,21 @@ describe('Parser', () => {
     });
 
     it('parses variables with whitespaces between name and ":"', () => {
-      let root = parse('@onespace : 42;');
+      const root = parse('@foo  : 42; @bar : 35;');
```

Comment:
as the test conditions have changed, please ad a new test with the new conditions and leave the old test intact